salt/addons/dns/init.sls

```diff
@@ -0,0 +1,26 @@
+{% if salt['pillar.get']('addons:dns', 'false').lower() == 'true' %}
+
+include:
+  - kube-apiserver
+  - kubectl-config
```

Comment:
I wonder, can the macro do the include?

Reply:
It could, but then there could be a problem if the macro is invoked twice... I would prefer to keep it as it is.